palette/handler.go

```diff
@@ -39,7 +38,7 @@ type Handler struct {
 func NewHandler(p *player.Player) *Handler {
 	h := &Handler{p: p, close: make(chan struct{})}
```

Comment:
Would remove the player from this handler too. (Or change it to an EntityHandle if necessary).